server/routes/api/follow.js

```diff
@@ -0,0 +1,12 @@
+import express from 'express';
+import FollowController from '../../controllers/FollowController';
+import Authenticate from '../../middlewares/Authenticate';
+
+const follow = express.Router();
+
+follow.post('/profiles/:username/follow', Authenticate.auth, FollowController.followUser);
+follow.delete('/profiles/:username/unfollow', Authenticate.auth, FollowController.unfollowUser);
+follow.get('/profiles/:userId/following', Authenticate.auth, FollowController.getAllUserIFollow);
```

Comment:
Yes both are unique entities but I choose to user userId because of the way I related the User table with the Follow table. The reference used here is the userId.

Reply:
Is it possible for you to get and make use of the username instead so as to promote uniformity on the routes?

Reply:
The relationship is not link by the username but by the userId so to say username attribute does not exist in the follolw table